.github/workflows/release.yml

```diff
@@ -112,4 +112,4 @@ jobs:
             conventional-changelog-conventionalcommits@^5.0.0
             @semantic-release/git@^10.0.1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.RELEASE_PAT }}
```

Comment:
what are the actual permissions needed for this PAT? It might be good to document or comment it, in case we need to replace it someday.